### DISCUSSION TITLE
Sensitive metadata?

### DISCUSSION THREAD

**Discussion by SebastianStehle** [SixLabors/ImageSharp#2492] [Q&A] [answered]
Hi,
I got the feature request to delete sensitive metadata from images, when they are uploaded. And indeed, this makes sense.

I already remove the orientation and auto rotate the image and I should also remove GPS related tags. But is there other metadata that is sensitive and should be removed or can I just remove all profiles? Obviously the orientation is important and I am curious if there is something similar.

**Comment by JimBobSquarePants** [comment, marked as answer]
I think you can remove everything tbh. It's supplementary information following decode and our encoders will fill in anything missing.

**Reply by SebastianStehle** [reply]
Thx. Exception would be orientation, if you don't autofix that.

**Reply by JimBobSquarePants** [reply]
Yeah, sorry, I meant after reorientating the image.

**Reply by JimBobSquarePants** [reply]
If converting to webp at any point it's actually recommended to use the auto orienting functionality as browser support for it with webp files is terrible. https://zpl.fi/exif-orientation-in-different-formats/

**Reply by SebastianStehle** [reply]
Yes...I do that already. I just renamed my method and also remove all metadata now. BTW. I am offloading all this to a microservice, because I realized that it hurts my general web performance to have such high cpu tasks in between.
